lib/azure/resource_management/vnet_config.rb

```diff
@@ -0,0 +1,254 @@
+#
+# Author:: Aliasgar Batterywala (aliasgar.batterywala@clogeny.com)
+#
+# Copyright:: Copyright (c) 2016 Opscode, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'ipaddress'
+
+module Azure::ARM
+  module VnetConfig
+
+    ## lists subnets of only a specific virtual network address space ##
+    def subnets_list_specific_address_space(address_prefix, subnets_list)
+      list = []
+      address_space = IPAddress(address_prefix)
+      subnets_list.each do |sbn|
+        subnet_address_prefix = IPAddress(sbn.properties.address_prefix)
+
+        ## check if the subnet belongs to this address space or not ##
+        list << sbn if address_space.include? subnet_address_prefix
+      end
+
+      list
+    end
+
+    def vnet_get(resource_group_name, vnet_name)
```

Comment:
Ruby best practices says to avoid get or set for getters and setters. So this is not really a getter or setter method so we could use method name like get_vnet which makes it more readable.